loggly/index.d.ts

```diff
@@ -1,23 +1,66 @@
-// Type definitions for loggly 1.0.8
-// Project: https://github.com/nodejitsu/node-loggly
-// Definitions by: Ray Martone <https://github.com/rmartone>
+// Type definitions for loggly 1.1
+// Project: https://github.com/winstonjs/node-loggly
+// Definitions by: Ray Martone <https://github.com/rmartone>, Joshua DeVinney <https://github.com/geoffreak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export interface LogglyOptions {
-    token: string;
-    subdomain: string;
-    tags?: string[];
-    json?: boolean;
-    host?: string;
-    auth?: {
-        username: string;
-        password: string;
+declare namespace Loggly {
```

Comment:
you do not need the namespace. just keep every thing on the top level with `export` and remove `export = ` at the end.